### ISSUE TITLE
Show completed jobs that are pending QA

### ISSUE THREAD

**Issue by carlosmiranda** [zerocracy/farm#911]
Right now, if a job is completed, it is removed from the Agenda page. If QA is available, that task will not be paid until QA is completed. However, we don't currently display completed jobs that are pending QA, which may make it difficult to keep track of which jobs are paid or not.

Let's add this info to a new section in the "Agenda" page (or somewhere else... let's just display it somehow). Once QA is complete the corresponding job should be removed.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#911](https://github.com/zerocracy/farm/issues/911) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by yegor256** [comment]
@carlosmiranda I think we should just add a job to the agenda of a performer, when we add it to the QA. And then remove it from there, where we remove it from the QA's agenda. The only question is what should be the status in the agenda, to differ them from normal jobs. Maybe just set `role` to `QA`?

**Comment by carlosmiranda** [comment]
@yegor256 better to have a separate status indicator. `QA` role is also in the agenda of the QA reviewer, so it would be confusing.

**Comment by yegor256** [comment]
@carlosmiranda OK, let's add `inspection` element, which will include the name of the inspector.

**Comment by 0crat** [comment]
The job [#911](https://github.com/zerocracy/farm/issues/911) assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1562022245); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by g4s8** [comment]
@0crat waiting for zerocracy/datum#347 to resolve

**Comment by 0crat** [comment]
> @0crat waiting for zerocracy/datum#347 to resolve ([here](https://github.com/zerocracy/farm/issues/911#issuecomment-395806031))

@g4s8 The impediment for [#911](https://github.com/zerocracy/farm/issues/911) was registered successfully by @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by g4s8** [comment]
@rultor release, tag=`0.23.3`

**Comment by g4s8** [comment]
@carlosmiranda it was implemented in #1254, you will see QA-inspector in agenda for new jobs after `0.23.3` release. Can we close?

**Comment by g4s8** [comment]
@carlosmiranda ping

**Comment by carlosmiranda** [comment]
@g4s8 thanks, missed this one sorry. Closing...

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
The job [#911](https://github.com/zerocracy/farm/issues/911) is now out of scope
